integration_tests/test_list_01.py

```diff
@@ -18,4 +18,20 @@ def test_list():
     for i in range(15):
         assert (f[i] - a[i]) == (i + 1.0)
 
-test_list()
+def test_list_02():
+    x: list[f64] = [1.0, 2.0]
+    y: list[f64] = []
```

Comment:
Let's make this test a little bit stronger. Doing appends to `x` in a loop after this and then do the actual checking.